dali/kernels/erase/erase_cpu.h

```diff
@@ -111,7 +127,7 @@ class EraseCpu {
     const T *in_ptr = in.data;
     T *out_ptr = out.data;
     if (out_ptr != in_ptr) {
-      std::memcpy(out_ptr, in_ptr, volume(shape) * sizeof(T));
+      detail::CopyImpl(out_ptr, in_ptr, volume(shape) * sizeof(T));
```

Comment:
Isn't this error for anything with `sizeof(T) > 1`?

Reply:
... ,right